3rdparty/toluapp/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@
 # Please note that the package source code is licensed under its own license.
 
 project ( toluapp C )
-cmake_minimum_required ( VERSION 3.4 )
+cmake_minimum_required ( VERSION 3.15 )
```

Comment:
3.15 was released in 2019 (says ChatGPT), so all targeted platforms should have it and it removes the annoying warning that 3.4 won't be supported.